### DISCUSSION TITLE
Media titles for whitewater kayaking

### DISCUSSION THREAD

**Discussion by dhajoglou** [raysan5/raylib#2402] [Show and tell]
I've been on this task for a while and finally made something I'm excited about!  This was on a Raspberry Pi 3 B+ and I used a java spring app with the JayLib java wrapper.  The box on the left is a countdown clock (45 second countdown) that's getting the data from a pubsub connection to a Redis Cache and the paddler name is sent via a REST api.  This is the first iteration where I will eventually build out heats, results, and a full race timing suite for downriver sprints.

I'm very happy with RayLib and very happy with the JayLib wrapper!

<img width="870" alt="Screen Shot 2022-03-21 at 5 23 06 PM" src="https://user-images.githubusercontent.com/32944638/159381440-a40b8c52-b423-46a6-ad58-77024f416324.png">

**Comment by raysan5** [comment]
@dhajoglou Wow! Very nice project! Congratulations!
